lib/federation-utils.js

```diff
@@ -23,7 +23,81 @@ var compareFederateKeys = (publicKeysA, publicKeysB) => {
   return comparison;
 };
 
+const getActiveFederationKeys = async (bridge) => {
```

Comment:
```suggestion
const getActiveFederationPublicKeys = async (bridge) => {
```

just to be consistent with the other name

Reply:
Updated.